docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_foreign_data_wrapper.md

```diff
@@ -0,0 +1,87 @@
+---
+title: ALTER FOREIGN DATA WRAPPER statement [YSQL]
+headerTitle: ALTER FOREIGN DATA WRAPPER
+linkTitle: ALTER FOREIGN DATA WRAPPER
+description: Use the ALTER FOREIGN DATA WRAPPER statement to alter a foreign-data wrapper.
+menu:
+  latest:
+    identifier: ddl_alter_foreign_data_wrapper
+    parent: statements
+aliases:
+  - /latest/api/ysql/commands/ddl_alter_foreign_data_wrapper/
```

Comment:
Aliases shouldn't be required for a new page... did you have a reason to add this?

```suggestion
```